docs/how-to-define-a-language.md

```diff
@@ -0,0 +1,7 @@
+# How Languages Are Defined
+
+1. Add language code to [../src/i18n/baseConfig.ts](/src/i18n/baseConfig.ts)
```

Comment:
Could you add an example of what a language code looks like? (Also a nit making it into a full sentence instead of just a bullet)

```suggestion
1. Add the new language code to [../src/i18n/baseConfig.ts](/src/i18n/baseConfig.ts)
```